devito/interfaces.py

```diff
@@ -223,6 +223,36 @@ def dyz(self):
         """Symbol for the cross derivative wrt the y and z dimension"""
         return cross_derivative(self, order=int(self.space_order/2), dims=(y, z))
 
+    @property
+    def dxl(self):
```

Comment:
Ok, cool. I just wanted to make sure that this is easily understandable for new users.